lib/puppet/provider/radius_server/cisco_nexus.rb

```diff
@@ -0,0 +1,108 @@
+require 'puppet/resource_api/simple_provider'
+require 'puppet_x/cisco/cmnutils'
+
+# Implementation for the radius_server type using the Resource API.
+class Puppet::Provider::RadiusServer::CiscoNexus < Puppet::ResourceApi::SimpleProvider
+  def canonicalize(_context, resources)
```

Comment:
In the unit tests could the `it_behaves_like 'a noop canonicalizer'` be added? 

https://github.com/puppetlabs/cisco-network-puppet-module/blob/puppet6/spec/unit/puppet/provider/banner/cisco_nexus_spec.rb#L96

Reply:
Added